_includes/code/quickstart.autoschema.endtoend.mdx

```diff
@@ -136,6 +136,86 @@ async function importQuestions() {
 importQuestions();
 ```
 
+</TabItem>
+<TabItem value="ts" label="TypeScript">
+
+```ts
+import weaviate, { WeaviateClient } from "weaviate-ts-client";
+import ObjectsBatcher from "weaviate-ts-client/types/batch/objectsBatcher"
```

Comment:
Once [this PR](https://github.com/weaviate/typescript-client/pull/32) is merged in, which will happen prior to the release, we can simply import `ObjectsBatcher` alongside the others:

```
import weaviate, { ObjectsBatcher, WeaviateClient } from 'weaviate-ts-client';
```

Reply:
Also small nit, but the above imports use double quotes, but the below `'node-fetch'` uses singles

Reply:
Perfect, fixed :)